Added option to add and update ExternalIdentifier in company endpoints (#387)
MewsSystems/gitbook-connector-api: changelog/README.md

# Changelog

> ## 23th January 2023

Reviewer: ```suggestion
## 26th January 2023
```

> * Added external identifier to [Add companies](../operations/companies.md#add-companies) and [Update companies](../operations/companies.md#update-companies)

Reviewer 2: More unified with our approach imho

```suggestion
* Extended [Add companies](../operations/companies.md#add-companies) and [Update companies](../operations/companies.md#update-companies) with `ExternalIdentifier`.
```
